java/code/src/com/suse/manager/webui/templates/minion/minion-header.jade

```diff
@@ -18,6 +18,6 @@
 if h.formulaValueEquals(server, "system-lock", "minion_blackout", "true") and not h.hasSshPushContactMethod(server)
   .alert.alert-warning
     p.
-      System is locked. Some operations are not allowed.
+      System is locked. Some actions are not allowed.
```

Comment:
Doc (and release notes) says:

> When you have added the {caasp} nodes to {productname}, the registered system will be locked automatically. When a system is locked, the {webui} shows a warning and you can schedule actions using the {webui} or the API, but the action will fail.

Is it "some" actions or "all actions"? What the doc and the release notes say is, FMPOV: All actions.

However it seems the formula says otherwise: https://github.com/SUSE/salt-formulas/pull/42#pullrequestreview-371436572

So it's "all", "all except", or "some"?